UI/GameSettingsScreen.cpp

```diff
@@ -576,8 +576,8 @@ void GameSettingsScreen::CreateViews() {
 		return UI::EVENT_CONTINUE;
 	});
 #if defined(USING_WIN_UI)
-	controlsSettings->Add(new ItemHeader(co->T("Mouse", "Mouse Control Settings")));
-	CheckBox *mouseControl = controlsSettings->Add(new CheckBox(&g_Config.bMouseControl, co->T("Use Mouse Control")));
+	controlsSettings->Add(new ItemHeader(co->T("Mouse", "Mouse settings")));
+	CheckBox *mouseControl = controlsSettings->Add(new CheckBox(&g_Config.bMouseControl, co->T("Use mouse control")));
```

Comment:
This changes the translation key so you'll have to update all the inis. Instead you can make the same tweak in en_US.ini.  Yeah, this is annoying..